A32NX/html_ui/Pages/A32NX_Core/A32NX_Core.js

```diff
@@ -12,7 +12,8 @@ class A32NX_Core {
             new A32NX_FWC(),
             new A32NX_GPWS(this),
             new A32NX_GroundReference(),
-            new A32NX_Speeds()
+            new A32NX_Speeds(),
+            new A32NX_Warning
```

Comment:
There is no reason to introduce a new warning system, this logic should be handled by the Flight Warning Computer (A32NX_FWC). I suggest moving your code to A32NX_FWC and removing A32NX_Warning.